src/utils.ts

```diff
@@ -66,19 +66,13 @@ export const absolutePosition = (el: Element): FocusOverlayPosition => {
   let top = 0;
   let width = 0;
   let height = 0;
-  const offsetBase: HTMLElement = document.createElement('div');
```

Comment:
what's the reason of removing the offset element?

Reply:
It is not necessary to do it like that anymore. a simple `document.body.getBoundingClientReact()` suffices in that case. 

Have a look at this answer: https://stackoverflow.com/a/32623832/8862005

> The above code can be simplified a lot by just using document.body.getBoundingClientRect() instead of adding a div

or here: https://stackoverflow.com/a/11396681/14865078